lib/pharos/addons/ingress_nginx.rb

```diff
@@ -38,7 +38,8 @@ def install
         apply_stack(
           configmap: config.configmap || {},
           node_selector: config.node_selector || {},
-          image: image_name
+          image: image_name,
+          worker_hosts: addon_manager.config.worker_hosts.size
```

Comment:
The addons already have `config.worker_hosts.size`, no need to pass the `addon_manager`

Reply:
Addon's `config` is the struct/schema thing for the current addon:

```ruby
config = configs[addon_class.name]
```

Reply:
Bleh, right. The addons should probably also get the top-level config... passing the `addon_manager` doesn't feel right.